qCC/ccEntityAction.cpp

```diff
@@ -787,7 +788,7 @@ namespace ccEntityAction
 
 				dlg.doubleSpinBox1->setStatusTip(QObject::tr("3*sigma = 99.7% attenuation"));
 				dlg.doubleSpinBox2->setStatusTip(QObject::tr("Scalar sigma controls how much the filter behaves as a Gaussian Filter\nSigma at +inf uses the whole range of scalars"));
-				dlg.doubleSpinBox3->setStatusTip(QObject::tr("It will only use colors for averaging where all of color components in the range of [threshold:255-threshold]"));
+				dlg.doubleSpinBox3->setStatusTip(QObject::tr("It will only use colors for averaging where all of the color components in the range of [threshold:255-threshold]"));
```

Comment:
```suggestion
				dlg.doubleSpinBox3->setStatusTip(QObject::tr("For averaging, it will only use colors for which all components are in the range [threshold:255-threshold]"));
```